### PR TITLE
fix(locale): dateFns generator for korean

### PR DESCRIPTION
**Pull request by nnnnoel**

### CODE REVIEW

**Comment by nnnnoel** [comment]
@zombieJ can you review about this PR?

**Comment by yoyo837** [comment]
Could you talk about the problems and background of this PR fix?

**Comment by nnnnoel** [comment]
@yoyo837 date-fns package do not have `ko_KR` or `koKR` fallback, only `ko`

**Comment by yoyo837** [comment]
> @yoyo837 date-fns package do not have `ko_KR` or `koKR` fallback, only `ko`

Could your add some test case for it?

**Comment by nnnnoel** [comment]
@yoyo837 Test Added

**Comment by yoyo837** [comment]
test failed.

**Comment by nnnnoel** [comment]
@yoyo837 fixed.

File: tests/generate.spec.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import luxonGenerateConfig from '../src/generate/luxon';
 import { getMoment } from './util/commonUtil';
 
 import 'dayjs/locale/zh-cn';
+import 'dayjs/locale/ko';
```
Comment (MadCcc) [inline tests/generate.spec.tsx:9]: Seems change in `generate/dateFns.ts` should not affect dayjs, is this import necessary?

Comment (nnnnoel) [reply tests/generate.spec.tsx:9]: dayjs locale js file should be imported before use.
cause of dynamic register locale config

Comment (yoyo837) [reply tests/generate.spec.tsx:9]: The problem is that the fix for dateFNS should not be covered using dayjs test cases.

**Comment by nnnnoel** [comment]
@MadCcc @zombieJ @yoyo837 could you please review that PR?

**Comment by yoyo837** [comment]
> @MadCcc @zombieJ @yoyo837 could you please review that PR?

Your test case looks like it has nothing to do with `date-fns`.

**Comment by nnnnoel** [comment]
@yoyo837 [this code](https://github.com/react-component/picker/pull/603/files#diff-fd7c88bad5cef16ec26bf210c809f964bfdbc294224cddb4be903a671637d430R24) tests all types of generator. I passed korean generator for all libraries.

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**